src/track/beatmap.cpp

```diff
@@ -574,6 +574,24 @@ void BeatMap::scale(enum BPMScale scale) {
         // remove every second third and forth beat
         scaleFourth();
         break;
+    case FOURTHIRDS:
+        if (getBpm() * 4 / 3 > getMaxBpm()) {
```

Comment:
I think you can drop these checks (and the check on double)... I'm not sure why we have a max BPM setting for anything other than determining when the BPM detector has gone awry.